src/store/state/state-model.ts

```diff
@@ -135,7 +135,11 @@ export const stateModel: StateModelIF = {
   feePrices: [cloneDeep(EmptyFees)],
   createResolution: {
```

Comment:
This name is historical: it was taken from Create UI, in which this data is actually "created". Here I think it makes more sense to just call it what it is: special resolution.